### ISSUE TITLE
Codeboxes are scrollable horizontally

### ISSUE THREAD

**Issue by MicLeey** [ory/web#454]
### Preflight checklist

- [X] I could not find a solution in the existing issues, docs, nor discussions.
- [X] I agree to follow this project's [Code of Conduct](https://github.com/ory/web/blob/master/CODE_OF_CONDUCT.md).
- [X] I have read and am following this repository's [Contribution Guidelines](https://github.com/ory/web/blob/master/CONTRIBUTING.md).
- [ ] This issue affects my [Ory Cloud](https://www.ory.sh/) project.
- [X] I have joined the [Ory Community Slack](https://slack.ory.sh).
- [X] I am signed up to the [Ory Security Patch Newsletter](https://ory.us10.list-manage.com/subscribe?u=ffb1a878e4ec6c0ed312a3480&id=f605a41b53).

### Describe the bug

Not sure if this is a bug or feature, but where code boxes used to wrap, they are now scrollable horizontally 

![image](https://user-images.githubusercontent.com/15386836/144448218-3ce2f072-5256-4d03-9142-0ee5be5066a4.png)

If this is intended, we should ensure the scrollbar appears at the bottom (e.g. playing with the height)
Imo we should shorten some of the URLs in there for explanatory value instead of making them hard to parse for correctness sake

### Reproducing the bug

https://www.ory.sh/open-source

### Version

N/A

**Comment by jfcurran** [comment]
this was changed with #431 and the change made was **[here](https://github.com/ory/web/pull/431/commits/436c4b902b516896b70732b0405468916a85382c)** - its a bug, caused by a necessary change elsewhere

**Comment by vinckr** [comment]
Oh dang. 
weirdly, I don't have the issue on Firefox. 
Maybe we should redo the whole code box component 🤔?

**Comment by MicLeey** [comment]
> weirdly, I don't have the issue on Firefox.
Strange, I do

I'll look into either making the code boxes we do have either wrap nicely or make sure values have a character limit

**Comment by jfcurran** [comment]
nice idea with the character limit - we can also just make sure code portions aren't unnecessarily complex or lengthy when they don't have to be. replacing lengthy strings with 'www.exampleurl.com' would be a great start anyways
